linksharing/handler.go

```diff
@@ -35,13 +39,29 @@ type HandlerConfig struct {
 
 	// Templates location with html templates.
 	Templates string
+
+	// TxtRecordTTL is the duration for which an entry in the txtRecordCache is valid.
+	TxtRecordTTL time.Duration
+}
+
+type txtRecord struct {
+	access    string
+	root      string
+	timestamp time.Time
```

Comment:
maybe timestamp should be `expiration`? or what's the timestamp for?

Reply:
okay it's when it was loaded. personally i think it might be easier to reason about if this value represents the timestamp when the entry expires. then you can do different expirations for different TXT records

Reply:
I will do this in a following PR

Reply:
:+1: